File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Javascript/Excercises/program-3.js
// 3-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 42.


let userInput = parseInt(prompt(' Ingresa un numero entero: '))


if (userInput % 42 == 0) {
    document.write(true)
    document.write(`\n, El numero ${userInput} es multiplo de 42`)
} else {
    document.write(false)
    document.write(`\n, El numero ${userInput} no es multiplo de 42`)
}

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Python/Excercises/program-3.py
# 3-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 42.


userInput = int(input('Ingresa un numero entero: '))

if userInput % 42 == 0:
    print(True)
    print(f'El numero {userInput } es multiplo de 42. ')
else:
    print(False)
    print(f'El numero {userInput } no es multiplo de 42. ')
